main/management/commands/kafka_consumer.py

```diff
@@ -70,23 +69,12 @@ def from_ers_message(message: Any) -> DruncMessage:  # type: ignore [misc]
     # Convert Kafka timestamp (milliseconds) to datetime (seconds).
     time = datetime.fromtimestamp(message.timestamp / 1e3, tz=timezone.utc)
 
-    try:
-        # TODO: Package and module names to be defined.
-        from ers.issue_pb2 import IssueChain  # type: ignore [import-not-found]
-
-    except ModuleNotFoundError:
-        return DruncMessage(
-            topic=message.topic,
-            timestamp=time,
-            message="ERS message could not be processed.",
-            severity="ERROR",
-        )
-
     ic = IssueChain()
     ic.ParseFromString(message.value)
 
-    # TODO: Check if the value of this is valid.
-    severity = ic.final.severity.upper()
```

Comment:
Is both `FATAL` and `CRITICAL` returned from ERS? I wonder if it's worth just altering our model to `FATAL`, if that's what is used now.

Reply:
ERS only return FATAL, no critical. But, in principle, we have CRITICAL severity in normal Kafka messages (See line 27 above). We could change that one to FATAL, change in the model CRITICAL by FATAL, and everything will be simpler and consistent. I would say that is the way to go.

Reply:
That sounds good to me.

Reply:
OK, so I've made the changes in [d5e75aa](https://github.com/ImperialCollegeLondon/drunc_ui/pull/336/commits/d5e75aa7e1fb1902b2306fea05ce3bdb67d04c3e)